### PR TITLE
End points  for Dissemination Service 

### LINKED ISSUE #40: Create CRUD endpoints for dissemination details
Dissemination form will be stored in two parts. First dissemination details will be created. Next farmer/attendee details will be stored and they will be mapped with a particular dissemination.

Create endpoints for the same.

Refer
https://github.com/Scaler-Community-Projects/NGO-Farmer-Analysis/wiki/Dissemination-Form

### PR DESCRIPTION
**Pull request by bhatiaPankaj**
Fixes #40

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -105,3 +105,9 @@ dist
 
 # Ignore environement files
 .idea/
+
+# Ignore sendgrid environement
+sendgrid.env 
+/src/server/logs/
+package-lock.json
```
Comment (ritesh-pandey) [inline .gitignore:112]: We will need lock file to ensure every machine has same dependency tree.

File: .gitignore [hunk 2]
```diff
@@ -105,3 +105,9 @@ dist
 
 # Ignore environement files
 .idea/
+
+# Ignore sendgrid environement
+sendgrid.env
```
Comment (ritesh-pandey) [inline .gitignore:110]: Not part of current task.

File: src/server/package.json [hunk 3]
```diff
@@ -22,10 +22,9 @@
     "dotenv": "^8.2.0",
     "express": "^4.17.1",
     "helmet": "^3.23.1",
+    "knex": "^0.21.1",
```
Comment (ritesh-pandey) [inline src/server/package.json:25]: Can you give a brief description of this module and why do we need it?

Comment (bhatiaPankaj) [reply src/server/package.json]: > A lot files are deleted in this PR. This is not part of current task. This creates confusion while reviewing.
> 
> Please make changes to only those files which are part of task.

My forked repo was not updated and this PR picked up changes already present in the base branch. I have merged the base branch to my fork. Now it shows relevant changes only. Can we go ahead with this one?

**Pull request merged**